ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/SaveBody.kt

```diff
@@ -96,6 +133,17 @@ public val HttpResponse.isSaved: Boolean
  *
  * [Report a problem](https://ktor.io/feedback/?fqname=io.ktor.client.plugins.skipSavingBody)
  */
+@Deprecated(
+    "Skipping of body saving for a specific request is not allowed anymore.\n" +
+        "$SHARE_USE_CASE_MESSAGE\n" +
+        API_REMOVAL_MESSAGE
+)
+@Suppress("UnusedReceiverParameter")
 public fun HttpRequestBuilder.skipSavingBody() {
-    attributes.put(SKIP_SAVE_BODY, Unit)
+    LOGGER.warn("Skipping of body saving for a specific request is not allowed anymore. $SHARE_USE_CASE_MESSAGE")
```

Comment:
If I have a 4 GB request, why can't I skip saving it?

Reply:
It won't be saved if you're working with it as with a streaming request.

In fact, this function was never doing what it was supposed to do. Even if you were calling it on a request, saving was skipped in `SaveBodyPlugin` but then the request anyways was saved in `HttpStatement.fetchResponse`